### ISSUE TITLE
string on two line with continuation line

### ISSUE THREAD

**Issue by maxime645** [TypeCobolTeam/TypeCobol#1248]
**Describe the bug**
The way we allow the user to write a string on two lines with continuation line char is wrong and won’t compile.

**To Reproduce**
(Type)Cobol code that cause the bug : (if any)

The following code is recognized as a correct way to write a string on two lines.
```cobol
       IDENTIFICATION DIVISION.
       PROGRAM-ID. DVZF0OSM.
       data division.
       working-storage section.
       01 foo pic X(50).
       PROCEDURE DIVISION.
      
             move "CHAMPS 082 A 085 INTERDITS EN OUVERTURE"
      -        ""Texte supplémentaire"
      -        to foo
      
           goback
           .
       END PROGRAM DVZF0OSM.
```

If there is only one " at line 9, the diagnostic suggests to add a second ".

**Expected behavior**
The diagnostic should suggest to remove the final " at superior line and forbid the use of 
the code above.

**Comment by fm-117** [comment]
In fact this syntax is valid and is used in NIST test `NC215A.cbl` ([at line 50](https://github.com/TypeCobolTeam/TypeCobol/blob/d6a78b3ff7fc618d84b005c89a01e9cbe495160e/TypeCobol.Test/ThirdParty/Nist/NC215A.cbl#L49)). This is combination of a continuation line and escaped double-quote within a string literal delimited by double quotes. (!) To escape the `"` character in a string literal delimited by double-quotes, the character has to be doubled. So here the line break happens right between the two consecutive double-quotes characters and therefore is valid.

This is supported by IBM compiler _BUT_ it is reported as a warning in RDZ COBOL editor by real-time compilation.

We should close this issue and not change the current behavior.

**Comment by rooksdo** [comment]
See also #1594

**Comment by smedilol** [comment]
I've tested with IBM compiler and this syntax is not valid.
Real-time syntax check of RDZ is not accurate.

You need to test with Remote Syntax check or compile the program.

The example from NIST test `NC215A.cbl` ([at line 50](https://github.com/TypeCobolTeam/TypeCobol/blob/d6a78b3ff7fc618d84b005c89a01e9cbe495160e/TypeCobol.Test/ThirdParty/Nist/NC215A.cbl#L49)) is not the same.
In Nist, it's in  ENVIRONMENT DIVISION / CONFIGURATION SECTION for an alphabet statement.

Here's all errors reported when I compile it:
```cobol
       IDENTIFICATION DIVISION.
       PROGRAM-ID. DVZF0OSM.
       data division.
       working-storage section.
       01 foo pic X(50).
       PROCEDURE DIVISION.
      
             move "CHAMPS 082 A 085 INTERDITS EN OUVERTURE"
      *KO
      *IGYPS0088-S The "MOVE" statement was invalid.  Expected "TO", but
      * found """".  The statement was discarded.
      
      *
      *IGYPS0083-E An alphanumeric literal with a length of zero was found.
      *A length of one character was assumed.
      -        ""Texte supplementaire"
      *KO
      *IGYPS0006-E The first character of a continued alphanumeric
      * literal was not a quote. A quote was assumed
      -        to foo
      
      *KO
      *IGYPS0007-E An alphanumeric literal was not properly continued.
      *The end of the literal was assumed on the last source line.
           goback
           .
       END PROGRAM DVZF0OSM.
```

**Comment by fm-117** [comment]
Your sample is not the same because the ```move "CHAMPS 082 A 085 INTERDITS EN OUVERTURE"``` ends before column 72 therefore the double quote is interpreted as a delimiter and not an escape character.
Try this one :

```cobol
       IDENTIFICATION DIVISION.
       PROGRAM-ID. DVZZPGM2.
       DATA DIVISION.
       WORKING-STORAGE SECTION.
      
       01 syntax-ok PIC X(200) VALUE "This text is way too long to fit o
      -                        "n a single line !".
      
       01 foo PIC X(200).
      
       procedure division.
           move "Crazy NIST ""Syntax" to foo
      
           move "                                           Crazy NIST "
      -    ""Syntax" to foo
      
           goback
           .
       END PROGRAM DVZZPGM2.
```

**Comment by LANOYDO** [comment]
this issue is closed; see issue #1594 for continuation line
